### PR TITLE
Useless and redundant code

### PR DESCRIPTION
**Pull request by gengjiawen**
##### Checklist

- [ ] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

### CODE REVIEW

File: src/node_quic_session.cc [hunk 1]
```diff
@@ -3229,7 +3227,7 @@ int QuicClientSession::SendPendingData(bool retransmit) {
   Debug(this, "Sending pending data for client session");
   int err;
 
-  err = SendPacket();
+  SendPacket();
```
Comment (gengjiawen) [inline src/node_quic_session.cc]: I am not sure this change ? Should we check this err too ?

@jasnell @danbev

Comment (jasnell) [reply src/node_quic_session.cc]: Should generally always check but I'm planning to refactor this and much of the err return code handling in an upcoming iteration

Comment (gengjiawen) [reply src/node_quic_session.cc]: Okay, I will drop the commit then :)

**Comment by jasnell** [comment]
Landed!

**Pull request closed without merging**